lib/appsignal/rack/sinatra_instrumentation.rb

```diff
@@ -70,7 +70,13 @@ def call_with_appsignal_monitoring(env)
             transaction.set_error(env["sinatra.error"])
           end
           transaction.set_action_if_nil(action_name(env))
-          transaction.set_metadata("path", request.path)
+          path =
+            if Appsignal.config[:sinatra_sanitized_routes]
+              sanitized_route(env)
+            else
+              request.path
+            end
+          transaction.set_metadata("path", path)
```

Comment:
Perhaps we shouldn't send `path` at all when this config option is enabled? I'm not sure if there's value in sending the route as `path` -- the actions are already being grouped by the route, so filtering by this tag won't do anything interesting.

And then maybe we can name it `sinatra_filter_path`, which I think would express its purpose more clearly. After all, routes, as in action names, are _already_ sanitised in Sinatra by default.

Reply:
I'll have a look if I can add a `filter_metadata` option really quickly to accomplish the same thing for all instrumentations.